api/api/views/audio_views.py

```diff
@@ -72,7 +72,7 @@ def thumbnail(self, request, *_, **__):
         if not image_url:
             raise NotFound("Could not find artwork.")
 
-        return super().thumbnail(request, audio, image_url)
+        return super().thumbnail(request, audio, image_url, AUDIO_TYPE)
```

Comment:
`MediaView::thumbnail` doesn't appear to accept a fourth argument :thinking: Is this leftover from something else or should it be cleaned up?

```suggestion
        return super().thumbnail(request, audio, image_url)
```